src/index.ts

```diff
@@ -19,10 +20,10 @@ const log = logger(path.basename(__filename), {
   level: process.env.LOG_LEVEL || 'fatal',
 });
 
-const digest = (payload) => {
+const hash = (payload) => {
```

Comment:
We should eventually just update the `crypto-lib` packages so that we don't have to maintain this function in two different repos.